model/src/main/java/hu/blackbelt/judo/meta/jsl/validation/JslDslValidator.xtend

```diff
@@ -1087,19 +1087,19 @@ class JslDslValidator extends AbstractJslDslValidator {
     def checkTransferField(TransferFieldDeclaration field) {
         try {
             if (field.maps !== null && !TypeInfo.getTargetType(field).isCompatible(TypeInfo.getTargetType(field.maps))) {
-                error("Type mismatch. Mapping expression value does not match field type.",
+                error("Type mismatch. Mapping expression value does not match field type at '" + field.name + "'.",
                     JsldslPackage::eINSTANCE.transferFieldDeclaration_Maps,
                     TYPE_MISMATCH)
             }
 
             if (field.reads !== null && !TypeInfo.getTargetType(field).isCompatible(TypeInfo.getTargetType(field.reads))) {
-                error("Type mismatch. Read expression value does not match field type.",
+                error("Type mismatch. Read expression value does not match field type at '" + field.name + "'.",
                     JsldslPackage::eINSTANCE.transferFieldDeclaration_Reads,
                     TYPE_MISMATCH)
             }
 
             if (field.isIsMany && !(field.referenceType instanceof TransferDeclaration)) {
-                error("Invalid collection of primitives.",
+                error("Invalid collection of primitives at '" + field.name + "'.",
```

Comment:
If a field type is not TransferDeclaration, how is it automatically primitive? I mean, can we rephrase this error message to actually state that if its a collection its referenced type is expected to be TransferDeclaration?